### ISSUE TITLE
conduit dashboard does not use the port provided to open the dashboard url when -p is used

### ISSUE THREAD

**Issue by leopoldodonnell** [runconduit/conduit#359]
Sorry to pile on with this minor dashboard command defect...

What happened:
```bash
$ conduit dashboard --port 8081
Running `kubectl proxy -p 8081`
Starting to serve on 127.0.0.1:8081

Opening [http://127.0.0.1:8001/api/v1/namespaces/conduit/services/web:http/proxy/] in the default browser
```

What I expected:
```bash
$ conduit dashboard --port 8081
Running `kubectl proxy -p 8081`
Starting to serve on 127.0.0.1:8081

Opening [http://127.0.0.1:8081/api/v1/namespaces/conduit/services/web:http/proxy/] in the default browser
```

**Comment by klingerf** [comment]
Whoa, nice catch @leopoldodonnell! We'll fix this asap, and we're also investigating better ways to handle the proxy port as part of #291.

**Comment by klingerf** [comment]
It appears that this functionality was originally broken in #53, and fixed in #249. The fix from #249 is going to ship with the next release, and in the meantime I'm going to close this issue.
